dex/testing/firo/electrumx.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+#
+# Set up ElectrumX-Firo (electrumX) regtest server for testing.
+# - Expects the firo regtest chain harness to be running and listening
+#   for RPC's on harness.sh ALPHA_RPC_PORT="53768"
+# - Exposes default RPC port at localhost:8000
+# - Exposes wallet SSL connection service at localhost:50002
+#
+# Requires:
+# - python3  - tested python3.10 and minimal testing python3.7
+# - pip3     - for boostrap loading pip
+# - gencerts - go install github.com/decred/dcrd/cmd/gencerts@release-v1.7
+# - git
+#
+# See Also: README_ELECTRUM_HARNESSES.md and README_HARNESS.md
+
+set -ex
+
+COMMIT=c0cdcc0dfcaa057058fd1ed281557dede924cd27
+
+ELECTRUMX_DIR=~/dextest/electrum/firo/server
+REPO_DIR=${ELECTRUMX_DIR}/electrumx-repo
+DATA_DIR=${ELECTRUMX_DIR}/data
+rm -rf ${DATA_DIR}
+mkdir -p ${REPO_DIR} ${DATA_DIR}
+
+cd ${REPO_DIR}
+
+if [ ! -d "${REPO_DIR}/.git" ]; then
+    git init
+    git remote add origin https://github.com/firoorg/electrumx-firo.git
```

Comment:
Fine, but would be way better if you guys updated the electrumx that everyone else uses.

https://github.com/spesmilo/electrumx/blob/c689a831cb4f3a8fc964424d7a77fe47df7d0377/electrumx/lib/coins.py#L2567-L2569

Reply:
Understand; that one is not my call.